src/classes/InstallScript.cls

```diff
@@ -29,9 +29,10 @@
 */
 public without sharing class InstallScript implements InstallHandler {
     private static final String EMAIL_REPLY_TO = 'donotreply@salesforce.org';
-    private static final String EMAIL_TO = 'jennifer.bennett+errors@salesforce.com';
+    private static final String EMAIL_TO = 'errors@salesforcefoundation.org';
     private static final String EMAIL_SUBJECT = 'V4S Install Errors';
     private static final String EMAIL_SENDER = 'V4S Install Script';
+    private static final String EMAIL_MESSAGE = 'Org Id: {0} encountered the following error: {1}';
```

Comment:
@katherinemitchell - This email message will get sent to the lead developers when an error occurs attempting to send the telemetry and will not be seen by the end user. Let me know if you think the wording should be changed. The {0} will be swapped out with the org Id that encountered the error and the {1} will be whatever error is caught. Thanks!

Reply:
'Org ID {0} encountered this error: {1}'

Reply:
@jjbennett